assets/.eslintrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "@thoughtbot/eslint-config",
+  "env": {
```

Comment:
I’m unfamiliar with `env` and `parserOptions` (below)…should they be a part of the shared config?

Reply:
They shouldn't be part of the shared config repo because not every project uses Javascript in the same environment, but may still wish to use the shared configs. The environment and parser options for Eslint are dependent on where you're running your code and how you're transpiling it (if at all). See [specifying environments](https://eslint.org/docs/user-guide/configuring#specifying-environments) and [specifying parser options](https://eslint.org/docs/user-guide/configuring#specifying-parser-options) for a good summary.